app/src/main/res/layout/item_create_new.xml

```diff
@@ -18,6 +18,7 @@ http://www.gnu.org/licenses
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:ignore="Overdraw"
     android:layout_width="match_parent"
```

Comment:
@pajatopmr your thoughts on using this as an approach to shutting off the overdraw warning?